src/translations/en-US.json

```diff
@@ -194,6 +194,16 @@
         "comment": "",
         "limit": 0
     },
+    "visualization.ErrorMessageMissingMapboxToken": {
+        "value": "Sorry, we can't display this insight",
+        "comment": "",
+        "limit": 0
+    },
+    "visualization.ErrorDescriptionMissingMapboxToken": {
+        "value": "An API access token is required to use Mapbox GL in Geo chart (pushpins)",
+        "comment": "",
```

Comment:
Here should be information - Mapbox GL should not be translated. Please, fill it out. Thanks. :)

Reply:
added. Thanks :)